project/src/main/java/com/google/sps/utilities/SpannerTasks.java

```diff
@@ -75,59 +88,83 @@ public static void insertorUpdateEvent(Event event) {
     SpannerClient.getDatabaseClient().write(mutations);
   }
 
-  /** Returns List of Event Ids from DB
+  /**
+   * Returns List of Event Ids from DB
    *
    * @param eventId List of IDs of event to be returned
-  */
+   */
   public static Set<Event> getEventsFromIds(List<String> eventIds) {
```

Comment:
Try to send one query with all ids?

The latency of one query includes:
  - Query to DB latency.
  - DB executing time.
  - Response to Service latency.

The current solution increases the 1st and 3rd latency.

Reply:
Done